kubernetes.go

```diff
@@ -34,8 +35,10 @@ type KubernetesService interface {
 	GetNodePool(ctx context.Context, clusterID, poolID string) (*KubernetesNodePool, *Response, error)
 	ListNodePools(ctx context.Context, clusterID string, opts *ListOptions) ([]*KubernetesNodePool, *Response, error)
 	UpdateNodePool(ctx context.Context, clusterID, poolID string, req *KubernetesNodePoolUpdateRequest) (*KubernetesNodePool, *Response, error)
+	// RecycleNodePoolNodes is DEPRECATED please use DeleteNode
```

Comment:
When we deprecate, let's give a schedule for removing it. For example:

```
// RecycleNodePoolNodes is DEPRECATED please use DeleteNode
// RecycleNodePoolNodes will be removed in version 2.0
```

(same comment for all deprecations)